src/styles/components/_component-card.scss

```diff
@@ -6,14 +6,14 @@
 
   .component-card {
     margin: 0;
-    border: 2px solid var(--foreground-color);
+    border: 2px solid var(--background-color);
     display: flex;
     flex-direction: column-reverse;
     height: 21.875rem; // 350px/16
     width: 100%;
     cursor: pointer;
     transition: all 150ms ease-in-out;
-    color: var(--font-color-dark);
+
 
     @media (prefers-color-scheme: dark) {
       border: 2px solid var(--background-color);
```

Comment:
Oh actually, I think I found an issue here: If I switch back and forth between light and dark mode, some of the styles are missing from one or the other. Here's what I did to fix it. 

This starts at about line 9 (to change the border color first): 
```
    border: 2px solid var(--background-color);
    display: flex;
    flex-direction: column-reverse;
    height: 21.875rem; // 350px/16
    width: 100%;
    cursor: pointer;
    transition: all 150ms ease-in-out;
    border: 2px solid var(--foreground-color);

    @media (prefers-color-scheme: dark) {
      border: 2px solid var(--background-color);
    }
```
That kept the black borders on light-mode, but then ALSO kept black borders on dark-mode.

Reply:
Thank you so much for figuring this one out!! :) :)